### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by MartinRosenberg** [taniarascia/takenote#135]
- [ ] Add a keyboard handler for each click handler (see [click-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md#click-events-have-key-events))
- [ ] Add a keyboard handler for each mouse movement handler (see [mouse-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md#mouse-events-have-key-events))
- [ ] Remove uses of `autoFocus` (see [no-autofocus](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-autofocus.md#no-autofocus))
- [ ] Add appropriate `role` (e.g. `button`) to any non-interactive HTML element (e.g. `div`) being used interactively (see [no-noninteractive-element-interactions](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md#no-noninteractive-element-interactions))
- [ ] Make sure user can tab back _out_ of the editor, after tabbing in
- [ ] Set `aria-grabbed` and `aria-dropeffect` for drag-and-drop
- [ ] Make settings modal closable via keyboard

**Comment by MartinRosenberg** [comment]
Work started in #135

**Comment by MartinRosenberg** [comment]
When you start dragging an element, you need to:

1. Set `aria-grabbed={true}` on that element
1. Visually highlight every meaningful drop target
1. Set `aria-dropeffect` on every meaningful drop target
    * `aria-dropeffect="link"` (or `"execute"`?) for categories/favorites
    * `aria-dropeffect="move"` for trash

#### See also

[This page](https://www.digitala11y.com/aria-dropeffect-properties/) about `aria-dropeffect`

**Comment by meowwwls** [comment]
> * [ ]  Add a keyboard handler for each click handler (see [click-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md#click-events-have-key-events))
> *

The thing to do in this case use use the proper HTML elements to begin with. If clicking it should do something (not take you somewhere), it should be a `button`. With a `button`, you get keyboard accessibility built in, not only with events but the ability to receive focus, and screenreader users know it's a button and performs an action without having to add and `aria-role`. Clicking that goes somewhere should be a link, but I think in this app it's pretty much all actions/buttons.

**Comment by taniarascia** [comment]
This is going on the backburner. If anyone wants to make the app more accessible, please feel free.
